src/__tests__/pages/guardian-directory.test.tsx

```diff
@@ -82,7 +82,7 @@ describe('GuardianDirectory', () => {
         )
         expect(screen.getByTestId(`${idx}_Rank`)).toHaveTextContent(value.Rank)
         expect(screen.getByTestId(`${idx}_DutyTitle`)).toHaveTextContent(
-          value.DutyTitle
+          value.DutyTitle.toUpperCase()
```

Comment:
Suggestion: I think we should handle this in the resolver for the personnel api, thoughts? https://github.com/USSF-ORBIT/ussf-personnel-api/blob/main/src/resolvers.ts

Reply:
Yeah you're right, not sure why I didn't think to do that. I'll make that change and get another PR up.

Reply:
Here is the PR: https://github.com/USSF-ORBIT/ussf-personnel-api/pull/30

Reply:
@gidjin I removed the check on this branch for the DutyTitle column, but I left the updated unit test. I'm thinking that the PR I linked above could be merged first?

Reply:
I'm looking at the other PR now :) thanks for the changes!